package.json

```diff
@@ -29,15 +29,15 @@
   },
   "homepage": "https://github.com/fastify/fastify-caching#readme",
   "devDependencies": {
-    "fastify": "^2.0.0",
+    "fastify": "^3.0.0-rc.1",
     "pre-commit": "^1.2.2",
     "snazzy": "^8.0.0",
     "standard": "^14.0.0",
     "tap": "^12.6.6"
   },
   "dependencies": {
     "abstract-cache": "^1.0.1",
-    "fastify-plugin": "^1.2.1",
+    "fastify-plugin": "^2.0.0",
     "uid-safe": "^2.1.5"
```

Comment:
Check for new versions of these.

Reply:
`tap` was the only outdated dependency, updated to the latest